src/battle/overlay_12_0224E4FC.c

```diff
@@ -3438,3 +3438,429 @@ int DamageDivide(int num, int denom) {
     
     return num;
 }
+
+int ov12_02253194(BattleSystem *bsys, BATTLECONTEXT *ctx) {
+    int i;
+    int j;
+    int script;
+    BOOL flag;
+    int battlerId;
+    int maxBattlers = BattleSystem_GetMaxBattlers(bsys);
+    
+    script = 0;
+    flag = FALSE;
+    
+    do {
+        switch (ctx->sendOutState) {
+        case 0: //field weather 
+            if (!ctx->weatherCheckFlag) {
+                switch (ov12_0223BAF8(bsys)) {
+                case 1:
+                case 2:
+                case 3:
+                    script = 271;
```

Comment:
should these be consts? unsure if we have a naix for it already or not

Reply:
I tried to look for one but couldn't find it. Lhea seems to know what the consts are but I'm unsure if pret has ported those yet